data/remnant missions.txt

```diff
@@ -495,3 +565,691 @@ mission "Remnant: Technology Available"
+			`	"Captain <last>! Thank you for your help in defending Viminal and transporting those wounded. I have transferred a small token of our appreciation to you. That being said, perhaps you would appreciate this a bit more: the other prefects and I have agreed to grant you access to a wider range of ships. Our capital class ships will now be available to you, including the Albatross and the Pelican. At the rate these raids are going, it is in everyone's best interests that you have access to a better warship. Please, meet us in the spaceport to further discuss the situation."`
+
+
+
+mission "Remnant: Salvage 4"
+	name "Track down the <npc>"
+	description "Track down and disable a Korath ship that escaped the raid on Viminal so that the Remnant may salvage it for technology."
+	source "Viminal"
+	to offer
+		has "Remnant: Salvage 3: done"
+	on offer
+		conversation
+			`Walking into the cafeteria you immediately sense the buoyant mood. According to a report on the video screen, very few Remnant ships were actually lost. Furthermore, of the ships that were lost, the escape pods worked perfectly, ensuring none of the crew were killed. Groups of crew are gathered around tables intermingled with the ground teams that keep their ships operational, and based on the snippets you can catch, they seem to be trading stories of crack shots and clever improvisations.`
+			`	Tali sits over at a corner booth with the prefect who met you at the dock and a couple other mechanics, and she waves you over. "Captain <last>! So, how do you like fighting with our fleet?" asks one of the mechanics.`
+			choice
+				`	"It seems odd to have a planetary defense fleet plundering."`
+				`	"It was a tough battle."`
+					goto next
+	
+			`	"I suppose it would," chants one of them. "We need access to their tech, though. Otherwise we'll fall behind." Judging by the finality of his statement, this is held to be common knowledge among the Remnant.`
+				goto next
+	
+			label next
+			`	While you're conversing with the group, the prefect suddenly glances down at his comlink, reads a message, then looks up at you. "It appears that a Korath ship survived the raid and has destroyed a few ships without taking cargo. Could you deal with it?"`
+			choice
+				`	"Sure."`
+				`	"Sorry, I don't have time to help you out."`
+					decline
+
+			`	"Thank you," he signs. "It is acting differently, jumping around destroying ships instead of looting and running. If it simply looted and left we would be pleased, but now it is learning about us instead, so it has reached the end of its usefulness. Please disable it so that our technicians can get on board." He pauses, then continues, "I doubt we'll be able to salvage the ship itself - they pack those things with soldiers - but we should be able to strip everything else of use off it."`
```

Comment:
first part changed to oddly.

Well, the Remnant consider it useful when the Raiders pick up cargo and take it back to wherever they come from. That helps ensure a steady stream of salvage for them to pick from. That is at least somewhat useful. At the very least, that is maintaining the cycle. 

That last part is something of a dig at the boarding and looting system. Even a starling can loot World Ships - so long as it doesn't get greedy and try to actually capture the thing. I picture it as being a process of just cutting holes in the ship from the outside to pull out equipment mounted in the outer perimeter of the ship. For that matter, a lot of the outfits are things that sound like they at least *might* be supposed to be installed on the outside. Some clearer are.

Reply:
That's correct, i suppose, but it doesn't explain why he says `so our technicians can get on board` - cutting holes in the hull is not being on board

Reply:
How about changing `Please disable it so that our technicians can get on board.` to `Please disable it so that our technicians can salvage parts from its exterior.` ?